assets/src/components/form/inputs/form-input.js

```diff
@@ -48,215 +48,206 @@ HTML5_INPUT_TYPES.push( 'iconButton' );
  * @param {Function} changeListener
  * @return {string} rendered date name form row
  */
-class FormInput extends Component {
-	static propTypes = {
-		type: PropTypes.oneOf( HTML5_INPUT_TYPES ).isRequired,
-		name: PropTypes.string.isRequired,
-		htmlId: PropTypes.string.isRequired,
-		htmlClass: PropTypes.string,
-		value: PropTypes.oneOfType( [
-			PropTypes.bool,
-			PropTypes.number,
-			PropTypes.string,
-		] ),
-		onChange: PropTypes.func,
-		dataSet: PropTypes.object,
-		options: PropTypes.array,
-		helpText: PropTypes.string,
-		helpTextID: PropTypes.string,
-		InputLayout: PropTypes.object,
-		changeListener: PropTypes.func,
-	};
-
-	render() {
-		const {
-			type,
-			name,
-			htmlId,
-			InputLayout,
-			changeListener = null,
-			afterInput = null,
-			...rest
-		} = this.props;
-		let {
-			helpTextID,
-			helpText = '',
-			htmlClass = '',
-			dataSet = {},
-			validations = [],
```

Comment:
these defaults are not being set anymore which could cause issues if code is expecting these values
this code be done using `propTypes` `defaultProps`